sdk/metricsadvisor/azure-ai-metricsadvisor/src/main/java/com/azure/ai/metricsadvisor/administration/MetricsAdvisorAdministrationClientBuilder.java

```diff
@@ -176,6 +176,7 @@ public MetricsAdvisorAdministrationAsyncClient buildAsyncClient() {
         return new MetricsAdvisorAdministrationAsyncClient(advisorRestAPIOpenAPIV2, serviceVersion);
     }
 
+    @SuppressWarnings("deprecation")
```

Comment:
What deprecated method is being used here?

Reply:
`HttpLogOptions.getApplicationId()` was deprecated in Core a while ago